scripts/exglobal_extractvars.sh

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash                                                                                                                                                                    
+
+################################################################################
+## UNIX Script Documentation Block
+## Script name:         exglobal_extractvars.sh
+## Script description:  Extracts variables from atmosphere, ocean, ice and wave
+##                      products and saves these variables in arcdir
+#######################
+# Main body starts here
+#######################
+
+source "${USHgfs}/preamble.sh"
+source "${USHgfs}/extractvars_tools.sh"
+
+# Scripts used
+EXTRCTVARA="${USHgfs}/atmos_extractvars.sh"
+EXTRCTVARO="${USHgfs}/ocnice_extractvars.sh"
+EXTRCTVARW="${USHgfs}/wave_extractvars.sh"
+
+# Set FHMAX_HF_GFS equal to FHMAX_GFS if FHMAX_HF_GFS is greater than FHMAX_GFS
+if (( FHMAX_GFS < FHMAX_HF_GFS )); then
+  export FHMAX_HF_GFS=FHMAX_GFS
+else
+  export FHMAX_HF_GFS=FHMAX_HF_GFS
```

Comment:
This is not needed; assigning the same value to itself.
```suggestion
else
  export FHMAX_HF_GFS=FHMAX_HF_GFS
```
```suggestion
```